.github/workflows/ci.yml

```
@@ -32,11 +32,19 @@ jobs:
       - run: yarn run test:system
       - run: yarn run generate-responses --silent
       - name: Build and install package
-        if: runner.os == 'Linux'
+        if: runner.os != 'Windows'
         run: |
           set -x
-          npx electron-builder --linux=deb
-          sudo apt install -y --no-install-recommends ./dist/installers/*.deb
+          case ${{ runner.os }} in
+            Linux )
+              npx electron-builder --linux=deb
```

```suggestion
              yarn electron-builder --linux=deb
```

This should be yarn now.

---

Done in 6f472611c330e592e3b121b04b161e176b23d697.